.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Thank you for making Rust GCC better!
+
+If your PR fixes an issue, you can add "Fixes #issue_number" into this
+PR description and the git commit message. This way the issue will be
+automatically closed when your PR is merged. If your change addresses
+an issue but does not fully fix it please mark it as "Addresses #issue_number"
+in the git commit message.
+
+Here is a checklist to help you with your PR.
+
+- \[ ] Read contributing guidlines
+- \[ ] `make check-rust` passes locally
+- \[ ] Run `clang-format`
+- \[ ] Added any relevant test cases to gcc/testsuite/rust.test
```

Comment:
Can you add an entry for copyright assignment?

Reply:
good point will do